### PR TITLE
Update/stats upgrade nudges styling

### PR DESCRIPTION
**Pull request by leogermani**
This PR implements the design of the Upgrade Nugdes that will be displayed in the Stats page. It also adds the description of each item and the "Learn more" links.

#### Changes proposed in this Pull Request:

* Style the Stats Upgrade nudges

#### Jetpack product discussion

p1HpG7-cfa-p2

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Activate and Connect Jetpack
* Edit your wp-config.php and add define( 'JETPACK_DEV_TEST_STATS_UPGRADE_NUDGES', true );
* Visit Jetpack > Site Stats
* Check the design against p6TEKc-56L-p2
* Check the icons
* Check the texts (check for typos)
* Check the links destinations

### CODE REVIEW

File: projects/plugins/jetpack/modules/stats/class-jetpack-stats-upgrade-nudges.php [hunk 1]
```diff
@@ -187,26 +226,36 @@ private static function get_upgrade_link( $source ) {
...
 					<img src="<?php echo esc_attr( $icon_url ); ?>" alt="" width="32" height="32">
 				</div>
 				<div class="dops-banner__content">
 					<div class="dops-banner__info">
-						<div class="dops-banner__title"><?php echo esc_html( $title ); ?></div>
+						<div class="dops-banner__title">
+							<?php echo esc_html( $title ); ?>
+							<p>
+								<?php echo esc_html( $text ); ?>
+								<a href="<?php echo esc_attr( $learn_more_link ); ?>">
```
Comment (kbrown9) [inline projects/plugins/jetpack/modules/stats/class-jetpack-stats-upgrade-nudges.php:254]: What do you think of opening the 'Learn more' links in a new tab?

Comment (kbrown9) [reply projects/plugins/jetpack/modules/stats/class-jetpack-stats-upgrade-nudges.php:254]: Also, do you think there would be any value to adding tracks ids to the learn more links?

Comment (leogermani) [reply projects/plugins/jetpack/modules/stats/class-jetpack-stats-upgrade-nudges.php:254]: > Also, do you think there would be any value to adding tracks ids to the learn more links?

We can track those clicks via the redirect service and I'm don't see any value in building funnels from them, so in my opinion we don't need it. But I'd ask @robertbpugh

Comment (leogermani) [reply projects/plugins/jetpack/modules/stats/class-jetpack-stats-upgrade-nudges.php]: > What do you think of opening the 'Learn more' links in a new tab?

Done

**Review by eeeeevon13 — COMMENTED** [review]
looks good. i'd like to follow up with an icon revision cause this is a heck of a lot of blue. will reply on the thread

**Review by kbrown9 — APPROVED** [review]
This looks good to me!

**Pull request merged**